src/content/docs/en/basics/layouts.mdx

```diff
@@ -236,39 +230,7 @@ const { title, fancyJsHelper } = Astro.props;
 <!-- -->
 ```
 
-<ReadMore>Learn more about Astro’s Markdown and MDX support in our [Markdown/MDX guide](/en/guides/markdown-content/).</ReadMore>
-
-### Using TypeScript with layouts
-
-Any astro layout can be modified to introduce typesafety & autocompletion by providing the types for your props:
-
-```astro ins={2-7} title="src/components/MyLayout.astro"
----
-interface Props { 
-	title: string;
-  description: string;
-  publishDate: string;
-  viewCount: number;
-}
-const { title, description, publishDate, viewCount } = Astro.props;
----
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="description" content={description}>
-    <title>{title}</title>
-  </head>
-  <body>
-    <header>
-      <p>Published on {publishDate}</p>
-      <p>Viewed by {viewCount} folks</p>
-    </header>
-    <main>
-      <slot />
-    </main>
-  </body>
-</html>
-```
+<ReadMore>Learn more about Astro’s Markdown and MDX support in our [Markdown guide](/en/guides/markdown-content/).</ReadMore>
```

Comment:
Just noting that it's an intentional choice to leave "and MDX" in here because *this* section only is about MDX. And, the Markdown guide is a good place to read about general Markdown (in MDX) stuff.